pilot/cmd/pilot-agent/options/options.go

```diff
@@ -98,6 +98,9 @@ var (
 	DNSCaptureAddr = env.RegisterStringVar("DNS_PROXY_ADDR", "localhost:15053",
 		"Custom address for the DNS proxy. If it ends with :53 and running as root allows running without iptable DNS capture")
 
+	DNSForwardParallel = env.RegisterBoolVar("ISTIO_META_DNS_FORWARD_PARALLEL", false,
```

Comment:
```suggestion
	DNSForwardParallel = env.RegisterBoolVar("DNS_FORWARD_PARALLEL", false,
```

and clear the model/context.go change

The istio_meta_ and NodeMetadata should only be needed when we are sending this data to Istiod. But this option only impacts the agent locally so no need

Reply:
Got it, thanks.